### PR TITLE
(Hopefully) Allows Unathi to wear clothes.

### PR DESCRIPTION
**Pull request by MechanicalVulpine**
## About The Pull Request

One line-change that adds Unathi (and Teshari) to the list of species allowed to wear clothing.

## Why It's Good For The Game

No more forced lizard nudity.

## Changelog
:cl:

fix: Lets Unathi wear clothing, hopefully.

/:cl:

(This is the first ever time I've done anything with Github, so this may not work.)

### CODE REVIEW

**Comment by pitaden** [comment]
Do teshari here have sprites for regular clothes?

**Comment by Redrover1760** [comment]
Kind of? I tested this locally with Teshari and it looks like jumpsuits just turn into weird grey jumpsuits, and nothing else seems to show up visually. (also it does let nonhumans wear clothing.)

However, I did run into a bug playing where, after signing up for a Surgeon (Civilian works fine) I straight up didn't even have an ID/phone. During the rounds hosted by Genesse, that is.

Edit: Actually, some clothes like labcoats and chemist jumpsuits work pretty well.

Funky as hell. 

Although, I still can't put on shoes... But at this point, who cares.

**Review by Redrover1760 — COMMENTED** [review]
Don't forget the Teshari also.

just add SPECIES_TESHARI and it works great.

**Review by Redrover1760 — CHANGES_REQUESTED** [review]
Don't forget Teshari also. (naked birb causes problems, like not having phones/ID if you are a doctor, for instance)

SPECIES_TESHARI

**Comment by genessee-forgot-his-password-again** [comment]
Test merged for Thursday. Ping me if you make commits after this so I can have the TM updated.

**Comment by genessee-forgot-his-password-again** [comment]
@MechanicalVulpine Please expand this PR to include Teshari. I don't really care if their sprites are jank; right now Teshari don't spawn with an ID, and I'd rather have IDed Tesh with weird sprites than IDless Tesh without.

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/item/clothing
 	name = "clothing"
 	siemens_coefficient = 0.9
-	var/list/species_restricted = list(SPECIES_HUMAN, SPECIES_HUMAN_VATBORN, SPECIES_HUMAN_CHILD, SPECIES_HUMAN_TEEN) //Only these species can wear this kit.
+	var/list/species_restricted = list(SPECIES_HUMAN, SPECIES_HUMAN_VATBORN, SPECIES_HUMAN_CHILD, SPECIES_HUMAN_TEEN, SPECIES_UNATHI) //Only these species can wear this kit.
```
Comment (genessee-forgot-his-password-again) [inline code/modules/clothing/clothing.dm]:
```suggestion
	var/list/species_restricted = list(SPECIES_HUMAN, SPECIES_HUMAN_VATBORN, SPECIES_HUMAN_CHILD, SPECIES_HUMAN_TEEN, SPECIES_UNATHI,SPECIES_TESHARI) //Only these species can wear this kit.

```

**Review by genessee-forgot-his-password-again — CHANGES_REQUESTED** [review]
Please include Teshari in this PR.

**Review by Redrover1760 — CHANGES_REQUESTED** [review]
You forgot a space between SPECIES_UNATHI and SPECIES_TESHARI. Put a space right before SPECIES_TESHARI

**Comment by pitaden** [comment]
Might be best to do `species_restricted = null` so it doesn't check for species at all, so you don't need to change the list each time a race gets added/removed

**Pull request merged**